tester_app/client/assets/images

```diff
@@ -0,0 +1 @@
+/Users/Rob/dev/open-source/react_on_rails-generator-results/tester_app/app/assets/images
```

Comment:
@robwise be sure you create the link by 

cd <destination directory>
ln -s ../../something

Then again, I was thinking maybe we can skip the sym links if we just have Rails have the proper directories in the asset paths. I haven't tried that.

Check with @alexfedoseev on the F&G setup.